hack/config-default.sh

```diff
@@ -9,7 +9,7 @@ image_pull_policy=${IMAGE_PULL_POLICY:-IfNotPresent}
 verbosity=${VERBOSITY:-2}
 package_name=${PACKAGE_NAME:-kubevirt-dev}
 push_log_file=${PUSH_LOG_FILE:-_out/imagePush.log}
-kubevirtci_git_hash="c7a1f8d14f722e911798f27bfc7511e94f6f2392"
+kubevirtci_git_hash="2de3c1108b4d52bf3e976cb828fc5fd6e6400603"
```

Comment:
LOL, I am surprised to see that this works, didn't know that you can reference commits of a pending PR, nice. I would have waited until the kubevirtci PR is merged and use the merge commit hash. And I would prefer to update this to that one, once the other PR is merged. Just to find it faster. But nice for testing WIP PRs :)

Reply:
I always thought of this as nothing else than a commit on a branch, accessible whether it is on a PR or not. Is there a setting to prevent this to work?

Reply:
Yeah, but the commit is on you clone still, not on kubevirtci yet, right? That is surprising to me. Maybe github does some magic here.
Thinking a second longer about it: it is in kubevirtci, because it has a git ref (`pull/<PRNR>`) to the commits of the PR. So it makes sense :slightly_smiling_face: